sdk/trace/provider.go

```diff
@@ -237,12 +258,15 @@ func (p *TracerProvider) ForceFlush(ctx context.Context) error {
 // Shutdown shuts down TracerProvider. All registered span processors are shut down
 // in the order they were registered and any held computational resources are released.
 func (p *TracerProvider) Shutdown(ctx context.Context) error {
+	if p.isShutdown.Load() {
```

Comment:
We might consider adding a comment that after `Shutdown` is called all other methods are No-Op.